docs/01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Tracing in Kyma
+---
+
+With the [Jaeger](https://github.com/jaegertracing) distributed tracing system, you can analyze the path of a request chain going through your distributed applications. This information helps you to, for example, troubleshoot your applications, or optimize the latency and performance of your solution.
+
+## Limitations
+
+In the production profile, Jaeger has no persistence enabled and keeps up to 10.000 traces stored in-memory. The oldest records are removed first.
+
+The evaluation profile has lower limits. For more information about profiles, see [Install Kyma: Choose resource consumption](../../../04-operation-guides/operations/02-install-kyma.md#choose-resource-consumption).
+
+
+## Benefits of distributed tracing
+
+Observability tools should clearly show the big picture, no matter if you're monitoring just a few or many components. In a cloud-native microservice architecture, a user request often flows through dozens of different microservices. Tools such as logging or monitoring help to track the request's path. However, they treat each component or microservice in isolation. This individual treatment results in operational issues.
+
+Distributed tracing charts out the transactions in cloud-native systems, helping you to understand the application behavior and relations between the frontend actions and backend implementation.
```

Comment:
Any non-idiomatic word for `chart out`? 🤔

Reply:
how about `tracks`?

Reply:
It makes it possible for you to identify and gasp the transaction by  correlating the involved components into one graph. That is a new aggregated information which you know can process but also visualize

Reply:
How about:
```suggestion
Distributed tracing in cloud-native systems aggregates the transaction data in one graph for processing and visualization. This helps to understand the application behavior and relations between the frontend actions and backend implementation.
```